wazuhapp/opensearch-dashboards/Docker/Dockerfile

```diff
@@ -8,7 +8,10 @@ RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-centostesting && \
     curl -sL https://rpm.nodesource.com/setup_10.x | bash - && \
     yum install nodejs git gcc gcc-c++ make sudo zip python3 -y && \
     alternatives --set python /usr/bin/python3 && \
-    npm install -g n 
+    npm install -g n
+
+RUN yum install -y \
```

Comment:
Any reason to keep this in a separate RUN command?

Reply:
Actually, no, I didn't see the other `yum install`, but this dependency is not necessary to build the package itself but to configure the packages names, I think it should be separately